xcell/mappers/mapper_KiDS1000.py

```diff
@@ -221,20 +183,17 @@ def _get_w2s2(self, mode):
         if self.w2s2s[mod] is not None:
             self.w2s2 = self.w2s2s[mod]
             return self.w2s2
-        lite, fn_lite = self._check_lite_exists(self.zbin,
-                                                f'{kind}_w2s2_ns{self.nside}',
-                                                True)
-        if lite:
-            print('Loading bin{} w2s2'.format(self.zbin))
-            self.w2s2s[mod] = hp.read_map(fn_lite)
-        else:
+
+        def get_w2s2():
```

Comment:
I'd make this function a method of the class (as in HSC, eg.)

Reply:
same reason as above. This depends on `mode` (also we're not losing any functionality and we can check the map through the rerun file if needed)